CMakeLists.txt

```diff
@@ -142,7 +143,7 @@ foreach(flag ${DEBUG_CXX_FLAGS})
 endforeach(flag)
 
 # Release flags
-foreach(flag -O3 -flto -DNDEBUG)
```

Comment:
Was there some issue with LTO?

Reply:
oh, there was error for previous versions. But it works fine now. I will remove this change.

Reply:
In fact we remove the flag because LTO does not work correctly with Clang